toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -137,6 +138,15 @@ func (s *SystemConfig) IsValid() (err error) {
 		return fmt.Errorf("invalid [KernelCommandLine]: %w", err)
 	}
 
+	// Validate that PackageRepos do not contain duplicate package repo name
+	repoNames := make(map[string]bool)
+	for _, packageRepo := range s.PackageRepos {
+		if _, ok := repoNames[packageRepo.Name]; ok {
```

Comment:
Also, call `packageRepo.IsValid()` for each. (See some of the other config files).

Reply:
Fixed